api/schemas/serializers.py

```diff
@@ -33,18 +33,13 @@ class RegistrationSchemaFormBlockSerializer(JSONAPISerializer):
 
     id = IDField(source='_id', read_only=True)
     type = TypeField()
-    page = ser.CharField(max_length=255, read_only=True)
-    section = ser.CharField(max_length=255, read_only=True)
-    help_text = ser.CharField(read_only=True, allow_blank=True)
-    block_id = ser.CharField(max_length=255)
+    answer_id = ser.CharField(max_length=255)
+    chunk_id = ser.CharField(max_length=255)
     block_type = ser.CharField(read_only=True)
-    block_text = ser.CharField(allow_blank=True)
-    size = ser.CharField(read_only=True, allow_blank=True)
-    choices = ser.ListField(
-        child=ser.CharField(read_only=True, allow_blank=True),
-        default=list(),
-    )
-    required = ser.BooleanField(default=True, read_only=True)
+    display_text = ser.CharField(read_only=True, allow_blank=True)
+    help_text = ser.CharField(read_only=True, allow_blank=True)
+    example_text = ser.CharField(read_only=True, allow_blank=True)
```

Comment:
Added `example_text` for prereg and OSF Preregistration.  For these schemas, `descriptions` are written for the purpose of the user completing the registration, and mapped to `help_text`.  `help` is really an example, for these schemas, so mapped to `example_text`.

Reply:
Alternatively, we could just do away with the example text altogether, not map it anywhere, for these 2 schemas.